canonical-web-and-design/snapcraft.io
# Replace members view with manage members view

## Done
Refactored the members view so that there is only a manage members view

## QA
- Go to https://snapcraft-io-3493.demos.haus/admin/ahnuP3quahti9vis8aiw/members
- Check that the members/invites tabs match [the design](https://app.zeplin.io/project/5f3170425822f41fc299a330/screen/606c2fda4c199f9300ed9372)
- *Ignore the "Add new member" button for now as it will be addressed in* https://github.com/canonical-web-and-design/snap-shttps://github.com/canonical-web-and-design/snap-squad/issues/1994

## Issue
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1992
